en/_posts/2016-02-28-using-timelion-with-laas.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title:  "Using Timelion with PaaS Logs"
+categories: Logs
+author: Babacar Diassé
+lang: en
+---
+
+Timelion is a Kibana plugin wich allows you to query multiple datasource at the same place like Grafana. This tool is very powerful to analyze metric contained in logs (or simply the logs count) sent to Elasticsearch (so PaaS Logs here). What you can do include but is not limited to :
```

Comment:
- typo: datasources
- "at the same place" do you mean "at the time" or "from a single Kibana instance" ?